src/components/CustomizedVacancyTable.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { Box, LinearProgress, Button, Avatar } from '@mui/material';
+import { NavLink } from 'react-router-dom';
+import { Applicant } from '../types/applicant';
+import { Skill } from '../types/skill';
+
+const StyledTableCell = styled(TableCell)(() => ({
+  [`&.${tableCellClasses.head}`]: {
+    backgroundColor: '#FFFFFF',
+    color: '#4d4d4d',
+    fontSize: 20,
+    border: 5,
+    boxShadow: 0,
+  },
+  [`&.${tableCellClasses.body}`]: {
+    fontSize: 16,
+    border: 5,
+    boxShadow: 0,
+  },
+}));
+
+const StyledTableRow = styled(TableRow)(() => ({
+  '&:nth-of-type(odd)': {
+    backgroundColor: '#f3f3f3',
+    boxShadow: 0,
+  },
+  // hide last border
+  '&:last-child td, &:last-child th': {
+    border: 5,
+    boxShadow: 0,
+  },
+}));
+
+export interface VacancyTableProps {
+  applicants: Applicant[];
+  receivingDate: string; // TODO: Change this later to Date once the backend and database are connected
+}
+
+export default function CustomizedTables(props: VacancyTableProps) {
+  const maxProgressBarValue = 10;
+  const applicants = props.applicants;
+  const receivingDate = props.receivingDate;
+
+  const linkStyle = {
+    textDecoration: 'none',
+  };
+
+  return (
+    <TableContainer component={Paper}>
+      <Table
+        sx={{ minWidth: '60%', maxWidth: '85%', flexDirection: 'row', boxShadow: 0 }}
+        aria-label="table with skills and their rating"
+        size="medium"
+      >
+        <TableHead>
+          <TableRow>
+            <StyledTableCell sx={{ paddingRight: 15 }}>Applicant</StyledTableCell>
+            <StyledTableCell sx={{ paddingRight: 15 }}>Rating</StyledTableCell>
+            <StyledTableCell>Skills</StyledTableCell>
+            <StyledTableCell>Application receipt</StyledTableCell>
```

Comment:
This shoul be 'Application received'. I think Liam made a small mistake when translating the mock up to english. In the german version it says 'Bewerbungseingang'.

```suggestion
            <StyledTableCell>Application received</StyledTableCell>
```

Reply:
😂